src/Translations.js

```diff
@@ -357,13 +358,12 @@ async function addDuplicatedTranslation({ original, language }) {
 
   async function untranslatedReferencesFound(untranslatedReferences) {
     const duplicate = removeExcludedReferences(original, untranslatedReferences.map(x => x._id))
-    const valid = (await validateDocument(duplicate, schema)).every(x => x.level !== 'error')
```

Comment:
Ik neem aan dat dit een bewuste keuze is

Reply:
Yes, clones worden gemaakt als niet gepubliceerde documenten. Je moet ze dus altijd eerst valid maken, voordat je ze kunt publishen. Dat is in mijn ogen wenselijker dan dat je geen clone kunt maken wanneer een document een verplichte referentie heeft, maar die nog niet vertaald is.